src/hotspot/cpu/x86/sharedRuntime_x86.cpp

```diff
@@ -97,11 +98,19 @@ jne    1b        \n\
     :"=t"(retval)
     :"0"(x), "u"(y)
     :"cc", "ax");
+  } else {
+    jdouble (*addr)(jdouble, jdouble) = (double (*)(double, double))StubRoutines::fmod();
+    jdouble dx = (jdouble) x;
+    jdouble dy = (jdouble) y;
+
+    retval = (jfloat) (*addr)(dx, dy);
+  }
   return retval;
 JRT_END
 
 JRT_LEAF(jdouble, SharedRuntime::drem(jdouble x, jdouble y))
   jdouble retval;
+  if (UseAVX < 1) {
```

Comment:
Does the new stub work for AVX=1 also or does it need AVX = 2 at the minimum?
Also good to check for UseFMA as well here.

Reply:
Added !UseFMA for condition for which x87 code is used.